blog/2021/08/DEQ.md

```diff
@@ -0,0 +1,389 @@
+@def rss_pubdate = Date(2021, 8, 18)
+@def rss = """ Implementing Deep Equilibrium Models in Julia... """
+@def published = "18 August 2021"
+@def title = "Composability in Julia: Implement Deep Equilibrium Models via Neural ODEs"
+@def authors = """Chris Rackauckas, Mike Innes, Yingbo Ma, Jesse Bettencourt, Lyndon White, Vaibhav Dixit"""  
+<!-- authors waiting to be updated -->
+
+<!-- Translations: [Traditional Chinese](/blog/2019/04/fluxdiffeq-zh_tw) -->
+
+In this blog post we will show you how to easily, efficiently, and
+robustly use steady state solvers with neural networks in Julia. This
+blog assumes certain familiarity with Neural ODE research.
+
+<!-- ![Flux ODE Training Animation](https://user-images.githubusercontent.com/1814174/51399500-1f4dd080-1b14-11e9-8c9d-144f93b6eac2.gif) -->
+
+The combination of differential equations and neural networks has aroused wide-spread interest
+since 2017. For more on the history of this field, please see the seminal work
+[Neural Ordinary Differential Equations](https://arxiv.org/abs/1806.07366) and our previous
+blog post [DiffEqFlux.jl – A Julia Library for Neural Differential Equations](https://julialang.org/blog/2019/01/fluxdiffeq/).
+This is a great example of interweaving machine learning with another field of research.
+In doing so, not only do we get to reap the benefits of centuries of research in the
+differential equations community, we also shift our perspective from the discrete to the
+continuous for machine learning, and from the explicit to the black-box for differential
+equations.
+
+What follows in recent years is a surge of algorithms expanding the frontiers of applying
+Neural ODEs to physical problems, as well as studies exploring the theoretical implications
+of this new-born area of research. For the former, one can refer to a flurry of works in the
+scientific machine learning (SciML) community, many of which based on the flexible structure
+of the Julia programming language. Among the latter direction, Deep Equilibrium Models (DEQ)
+stand out as an extension of Neural ODE theory, in particular making the notion of infinite-depth
+neural networks explicit in the algorithm. This blog post will be aimed at exploring the
+properties of DEQ models.
+
+(Note: If you are interested in this work and are an undergraduate or graduate
+student, we have [Google Summer of Code projects available in this area](/soc/ideas-page). This
+[pays quite well over the summer](https://developers.google.com/open-source/gsoc/help/student-stipends).
+Please join the [Julia Slack](http://julialang.org/slack/) and the #jsoc channel to discuss in more detail.)
+
+\toc
+
+
+## What is a Neural Ordinary Differential Equation?
```

Comment:
I would just delete this entire section. It's just rambling and doesn't actually describe neural ODEs.